libraries/AP_Math/location_double.cpp

```diff
@@ -31,7 +31,7 @@
  */
 void wgsllh2ecef(const Vector3d &llh, Vector3d &ecef) {
   double d = WGS84_E * sin(llh[0]);
-  double N = WGS84_A / sqrt(1 - d*d);
+  double N = WGS84_A / safe_sqrt(1 - d*d);
```

Comment:
This is wrong as we are doing double calculation and safe_sqrt will reduce precision